store/postgres/src/chain_store.rs

```diff
@@ -1715,6 +1778,12 @@ impl ChainStoreTrait for ChainStore {
             .await
     }
 
+    async fn clear_call_cache(&self, from: Option<i32>, to: Option<i32>) -> Result<usize, Error> {
+        let storage = self.storage.clone();
+        let conn = self.get_conn()?;
+        storage.clear_call_cache(&conn, from, to)
```

Comment:
We don't need to clone the store here:

```suggestion
        let conn = self.get_conn()?;
        self.storage.clear_call_cache(&conn, from, to)
```